.github/workflows/python.yml

```diff
@@ -0,0 +1,107 @@
+name: Python bindings
+on: [push, pull_request]
+
+env:
+  CIBW_ENVIRONMENT_WINDOWS: PATH="$PATH;C:\\Boost\\bin" LIB="$LIB;C:\\Boost\\lib" INCLUDE="C:\\Boost\\include" VSCMD_ARG_TGT_ARCH=""
+  CIBW_ENVIRONMENT_MACOS: MACOSX_DEPLOYMENT_TARGET=10.14
+
+  CIBW_BEFORE_ALL: bash {package}/bindings/python/tools/install_boost.sh {package}
+  CIBW_BEFORE_BUILD: bash {package}/bindings/python/tools/install_boost_python.sh {package}
```

Comment:
instead of installing and building (all of) boost. I would suggest you just get the source archive and set `BOOST_ROOT` to point to it. When building libtorrent (with boost-build) it will pick up the dependencies from boost and *only* build the parts of boost that are used. I would expect this to speed up the build significantly.

but, it's possible I'm misunderstanding something

Reply:
This could work. However, I think that if Boost.Python not installed and built globally, `auditwheel` could have problems detecting it and linking it inside wheel. Maybe this can be solved if just Boost.Python (and Boost.System if it is needed) is built and installed, while other libraries just copy their headers to global directory.